ipaclient/remote_plugins/schema.py

```diff
@@ -64,6 +64,13 @@
 }
 
 
+class BytesEncoder(json.JSONEncoder):
+    def default(self, obj):
```

Comment:
Both solutions are fine. The default function argument is a few lines less code. The json module doesn't have to create a new encoder class for every call and can use the default encoder instance instead.